### ISSUE TITLE
Show insertion_order on one-to-one relationships in console

### ISSUE THREAD

**Issue by seanonthenet** [hasura/graphql-engine#7919]
Being able to see and change the `insertion_order` value on one-to-one relationships in the console is important.

**Comment by mikejackowski** [comment]
following 👀

**Comment by jzstern** [comment]
this would be very helpful — been using [this workaround](https://github.com/hasura/graphql-engine/issues/2576#issuecomment-1336110273)

**Comment by stephane303** [comment]
Yes that would also indicated that this parameter exists and it could be important, I stuggled one day to make my one to one relationship work !
